hyperspy/io_plugins/bruker.py

```diff
@@ -821,12 +821,12 @@ def calc_real_time(self):
         """calculate and return real time for whole hypermap
         in seconds
         """
-        line_cnt_sum = np.sum(self.line_counter)
+        line_cnt_sum = np.sum(self.line_counter).astype(np.float64)
```

Comment:
I assume that ``astype`` is needed because ``line_counter`` was float32 to start with. Wouldn't it be better to use ``np.sum(self.line_counter, dtype=np.float64)`` instead then?

Reply:
Yes, indeed!
The original type was int32 and the reason why it worked on linux and not on windows is the following: 

on linux
```python
>>> np.array([0]).dtype
dtype('int64')
```

on windows:
```python
>>> np.array([0]).dtype
dtype('int32')
```